microcosm_eventsource/tests/stores/test_rollup.py

```diff
@@ -80,9 +104,10 @@ def test_count(self):
     def test_retrieve(self):
         rollup = self.store.retrieve(self.task2.id)
         assert_that(rollup, has_properties(
-            _event=self.task2_assigned_event,
+            _event=self.task2_started_event,
             _container=self.task2,
             _rank=1,
+            _assignee="Alice",
```

Comment:
Payoff: columns from rows that aren't ranked first can still be copied forward during the roll up process.